pyk/src/tests/test_kastManip.py

```diff
@@ -176,7 +176,7 @@ def test_ml_pred_to_bool(self):
     )
 
     def test_bool_to_ml_pred(self):
-        for name, unsafe, ml_pred_expected, bool_in in self.test_data_bool_to_ml_pred:
+        for name, _, ml_pred_expected, bool_in in self.test_data_bool_to_ml_pred:
             with self.subTest(name):
                 ml_pred_actual = bool_to_ml_pred(bool_in)
                 self.assertEqual(ml_pred_actual, ml_pred_expected)
```

Comment:
We should just remove the `unsafe` parameter from this test completely. I thought before it would be possible for these two to be invertible, but I don't think so, so we should probably remove the comment saying so, I've struck out the line-item on this issue: https://github.com/runtimeverification/k/issues/2801